### DISCUSSION TITLE
Smokeshow/Coverage tests failing after latest updates

### DISCUSSION THREAD

**Discussion by romabozhanovgithub** [fastapi/fastapi#10182] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
That's place where it fails:
https://smokeshow.helpmanual.io/142o4k163r5f4j1o1s3j/d_a44f0ac069e85531_test_openapi_examples_py.html
```

### Description

After latest update my [PR](https://github.com/tiangolo/fastapi/pull/10121) failing on smokeshow/coverage tests and I found that is same for other new PRs. Looks like someone need to fix it

### Operating System

Linux

### Operating System Details

Linux LST

### FastAPI Version

0.103.0

### Pydantic Version

2.2.1

### Python Version

3.11

**Comment by Sion99** [comment]
This happens to my PR #10176, too.
Automatically made docs looks good to me but it kept failing smokeshow/coverage tests.

**Comment by mdlam92** [comment, marked as answer]
https://github.com/encode/starlette/discussions/2261#discussioncomment-6880161

Pretty sure that the latest `anyio` release broke the `fastapi` `TestClient`
